app/src/main/java/com/github/sdpteam15/polyevents/model/database/remote/DatabaseInterface.kt

```diff
@@ -67,12 +66,17 @@ interface DatabaseInterface {
      */
     var userSettingsDatabase: UserSettingsDatabaseInterface?
 
+    /**
+     * The database used to handle query about to route
+     */
+    var routeDatabase: RouteDatabaseInterface?
+
     /**
      * Add an Entity to the data base
      * @param element The element that needs to be added in the database
      * @param collection The collection to which we want to add the given element
      * @param adapter The adapter converting the element into a HashMap recognised by the database
-     * @return An observer that will be set to true if the communication with the DB is over and no error
+     * @return An observer that will be set to the id if the communication with the DB is over and no error
```

Comment:
How can you return the ID if the return type is the Observable<Boolean> ?

Reply:
fail of method

Reply:
Observable<String> so the String is the id if all work else an empty String